### PR TITLE
Integrate Zuul CI/CD

### PR DESCRIPTION
**Pull request by lego963**
Replace Travis CI/CD with Zuul CI/CD pipeline

### CODE REVIEW

**Review by anton-sidelnikov — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # OpenTelekomCloud Docker Machine driver
-[![Build Status](https://travis-ci.org/opentelekomcloud/docker-machine-opentelekomcloud.svg)](https://travis-ci.org/opentelekomcloud/docker-machine-opentelekomcloud)
+[![Zuul Gate](https://zuul.eco.tsi-dev.otc-service.com/api/tenant/eco/badge?project=opentelekomcloud/docker-machine-opentelekomcloud&pipeline=check&branch=master)](https://zuul.eco.tsi-dev.otc-service.com/t/eco/builds?project=opentelekomcloud%2Fdocker-machine-opentelekomcloud)
```
Comment (outcatcher) [inline README.md]: The branch should be `devel`

**Review by outcatcher — CHANGES_REQUESTED** [review]

**Comment by lego963** [comment]
recheck

**Comment by lego963** [comment]
recheck

**Comment by lego963** [comment]
recheck

**Review by anton-sidelnikov — DISMISSED** [review]

File: .goreleaser.yml [hunk 2]
```diff
@@ -29,14 +29,25 @@ builds:
...
+      - goos: darwin
+        goarch: arm
+      - goos: freebsd
+        goarch: arm
+      - goos: windows
+        goarch: arm
     binary: '{{ .ProjectName }}'
 archives:
-  - format: tar.gz
+  - format: zip
```
Comment (outcatcher) [inline .goreleaser.yml]: Please make sure `zip` works with Rancher

**Comment by lego963** [comment]
recheck

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by outcatcher — DISMISSED** [review]

**Comment by lego963** [comment]
recheck

**Review by anton-sidelnikov — APPROVED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**